polkadot.json

```diff
@@ -7,68 +7,68 @@
+        "details": "Welcome to the world of Polkadot, a platform for blockchains and Web3 applications - created by Gavin Wood, one of the co-founders of Ethereum. It is a vision towards a free, open, and decentralized world computer."
       },
       {
         "id": 2,
         "imageUrl": "https://i.imgur.com/9BQhqPY.png",
         "title": "Building blocks of Polkadot",
-        "details": "Think of Polkadot like building with Legos - you can pick and choose the pieces you want to create something unique! Imagine building a hyper-optimized blockchain, called a parachain, with specific characteristics to fit your exact need! All this is possible with Polkadot!"
+        "details": "Polkadot is designed to solve 3 fundamental problems in the blockchain space: scalability, security, and interoperability. The Polkadot ecosystem is composed of many independent web3 applications, like blockchains and DApps. Imagine building or using hyper-optimized blockchains, called a parachains, designed to fit to your exact needs! All of this is possible with Polkadot!"
       },
       {
         "id": 3,
-        "imageUrl": "https://i.imgur.com/XbI361a.png",
-        "title": "Unleash Versatility",
-        "details": "Plus, because of how it's designed, Polkadot can do things other blockchains can't, like handle lots of transactions at once and connect with external blockchain systems too."
+        "imageUrl": "https://i.imgur.com/kdwrLg5.png",
+        "title": "Scalability solved",
+        "details": "Polkadot scales Web3 applications through parallelization and sharding. Transactions to different applications can be executed in parallel, and data from these applications are also hosted in their own blockchain. This allows applications to grow quickly without affecting the fees or speeds of other applications."
       },
       {
         "id": 4,
-        "imageUrl": "https://i.imgur.com/kdwrLg5.png",
-        "title": "Efficiency & customizability",
-        "details": "Polkadot makes blockchain transactions faster and smoother by enabling seamless collaboration between different blockchains. This boosts transaction speed and reliability, allowing for parallel processing and data transfer. With Polkadot, complex transactions are executed with ease, making it more efficient and scalable. Get ready for a seamless and exciting blockchain experience!"
+        "imageUrl": "https://i.imgur.com/XbI361a.png",
+        "title": "Unleash Versatility",
+        "details": "Plus, because of how it's designed, applications built on Polkadot can do things you can't on other blockchains. For example: enable zero transaction fees, scale up and down on-demand, and permissionlessly upgrade and evolve over time."
```

Comment:
```suggestion
        "details": "Plus, because of how it's designed, applications built on Polkadot can do things you can't do on other blockchains. For example: enable zero transaction fees, scale up and down on-demand, and permissionlessly upgrade and evolve over time."
```